website/content/docs/configuration/storage/index.mdx

```diff
@@ -40,3 +40,60 @@ storage "file" {
+store Vault data. However, cost is relative to where you run the external storage.
+
+### Integrated storage vs. Consul as Vault storage
+
+[HashiCorp Consul](https://www.consul.io/docs/intro) is a comprehensive
+multi-cloud service networking solution including service mesh, service
+discovery, and network infrastructure automation. Vault can leverage the
+Consul's [KV Store](https://www.consul.io/api-docs/kv) to persist Vault data.
+Prior to Vault 1.4, Consul was the recommended Vault storage.
+
+The table below highlights the differences between integrated storage and Consul
+in addition to the table above.
+
+|                     | Integrated Storage                                                                                           | Consul                                                                                                                                                                                                                                                                                                                                           |
+| ------------------- | ------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Deployment          | Vault cluster is all you need.                                                                               | Vault cluster & Consul cluster. <p />Use a dedicated Consul cluster for Vault storage, and it should not be used for other purposes (e.g., service discovery, service mesh). <p />See the [Vault with Consul Storage Reference Architecture](https://learn.hashicorp.com/tutorials/vault/reference-architecture#recommended-architecture) guide. |
+| Data location       | Data is on disk.                                                                                             | All data is in memory.                                                                                                                                                                                                                                                                                                                           |
+| System requirements | Storage-optimized high IOPS machines.                                                                        | Memory-optimized machines.                                                                                                                                                                                                                                                                                                                       |
```

Comment:
Since the Reference Architectures should be the source of truth for the system requirements, I would probably link to those here instead. Because when I read 'storage-optimized' or 'memory optimized' instances, that doesn't align with the general compute instances w/ high IOPS storage that are recommended in the Reference Architectures. I would love to get @daveadams thoughts on this too.

Reply:
Fixed  ([commit](https://github.com/hashicorp/vault/pull/13999/commits/ee6605f8dd6cbbfef76a4c28407d268ec891afaf))